### PR TITLE
Quiet logging of TrueTimeClient

### PR DESCRIPTION
**Pull request by rickpasetto**

### CODE REVIEW

File: Loop/Managers/TrustedTimeChecker.swift [hunk 1]
```diff
@@ -45,7 +49,7 @@ class TrustedTimeChecker {
     }
     
     private func checkTrustedTime() {
-        ntpClient.fetchIfNeeded { [weak self] result in
+        ntpClient.fetchIfNeeded(completion: { [weak self] result in
```
Comment (darinkrauss) [inline Loop/Managers/TrustedTimeChecker.swift:50]: [nit] Why the addition of the completion argument? I don't believe it is needed.

File: Loop/Managers/TrustedTimeChecker.swift [hunk 2]
```diff
@@ -36,6 +36,10 @@ class TrustedTimeChecker {
 
     init(_ alertManager: AlertManager) {
         ntpClient = TrueTimeClient.sharedInstance
+        ntpClient.logCallback = { [weak self] _ in
+            // Too chatty
+//            self?.log.debug("%@", $0)
```
Comment (darinkrauss) [inline Loop/Managers/TrustedTimeChecker.swift]: [nit] Remove commented out code?

**Review by darinkrauss — APPROVED** [review]
A couple of comments, but otherwise approved. 🎃

File: Loop/Managers/TrustedTimeChecker.swift (hunk 1 above)
Comment (rickpasetto) [reply Loop/Managers/TrustedTimeChecker.swift:50]: This is actually a new Xcode 12 compiler warning, believe it or not:
Screen Shot 2020-10-19 at 5 03 24 PM

File: Loop/Managers/TrustedTimeChecker.swift (hunk 2 above)
Comment (rickpasetto) [reply Loop/Managers/TrustedTimeChecker.swift]: I left it in there in case someone may want to see the logging.  I'll make the comment clearer...

**Pull request merged**